Remotion/Web/Development.WebTesting.UnitTests/Configuration/TestInfrastructureConfigurationTest.cs

```diff
@@ -31,29 +31,20 @@ public class TestInfrastructureConfigurationTest
     [Test]
     public void CreateFromWebTestConfigurationSection ()
     {
-      var webTestConfigurationSection = CreateWebTestConfigurationSection();
-      PrivateInvoke.InvokeNonPublicMethod(webTestConfigurationSection, "set_Item", "webApplicationRoot", "http://some.url:1337/");
-      PrivateInvoke.InvokeNonPublicMethod(webTestConfigurationSection, "set_Item", "screenshotDirectory", @".\SomeScreenshotDirectory");
-      PrivateInvoke.InvokeNonPublicMethod(webTestConfigurationSection, "set_Item", "closeBrowserWindowsOnSetUpAndTearDown", false);
-      PrivateInvoke.InvokeNonPublicMethod(
-          webTestConfigurationSection,
-          "set_Item",
-          "requestErrorDetectionStrategy",
-          _testRequestErrorDetectionStrategyAssemblyQualifiedName);
+      var webTestSettingsStub = new Mock<IWebTestSettings>();
+      webTestSettingsStub.Setup(_ => _.WebApplicationRoot).Returns("http://some.url:1337/");
+      webTestSettingsStub.Setup(_ => _.ScreenshotDirectory).Returns(@".\SomeScreenshotDirectory");
+      webTestSettingsStub.Setup(_ => _.CloseBrowserWindowsOnSetUpAndTearDown).Returns(false);
+      webTestSettingsStub.Setup(_ => _.RequestErrorDetectionStrategyTypeName).Returns(_testRequestErrorDetectionStrategyAssemblyQualifiedName);
```

Comment:
```suggestion
      var webTestSettingsStub = new Mock<IWebTestSettings>();
      webTestSettingsStub
          .Setup(_ => _.WebApplicationRoot)
          .Returns("http://some.url:1337/");
      webTestSettingsStub
          .Setup(_ => _.ScreenshotDirectory)
          .Returns(@".\SomeScreenshotDirectory");
      webTestSettingsStub
          .Setup(_ => _.CloseBrowserWindowsOnSetUpAndTearDown)
          .Returns(false);
      webTestSettingsStub
          .Setup(_ => _.RequestErrorDetectionStrategyTypeName)
          .Returns(_testRequestErrorDetectionStrategyAssemblyQualifiedName);
```

preferred format if it's longer than a short & simple setup/return.